docs/help/troubleshooting.md

```diff
@@ -5,7 +5,21 @@
 ### CUDA-related errors
 
 When installing CUDA, it may cause errors because of version conflicts.
-To resolve this, try either:
+To resolve this, try either one of the following methods:
+
+- Unhold all CUDA-related libraries and rerun the setup script.
+
+  ```bash
+  sudo apt-mark unhold  \
+    "cuda*"             \
+    "libcudnn*"         \
+    "libnvinfer*"       \
+    "libnvonnxparsers*" \
+    "libnvparsers*"     \
+    "nvidia*"
```

Comment:
```suggestion
    "nvidia*" \
    "python3-libnvinfer"
```

One more thing is held in https://github.com/autowarefoundation/autoware/blob/main/ansible/roles/tensorrt/README.md#manual-installation
`libnvinfer8 libnvonnxparsers8 libnvparsers8 libnvinfer-plugin8 libnvinfer-dev libnvonnxparsers-dev libnvparsers-dev libnvinfer-plugin-dev python3-libnvinfer`

Reply:
@xmfcx Thank you. But we didn't install the python library with Ansible because we only use C++.
https://github.com/autowarefoundation/autoware/blob/main/ansible/roles/tensorrt/tasks/main.yaml

I think it's good to make the Ansible script and manual installation steps consistent.
Should we add it to the Ansible script or should we remove it from the manual installation steps?

@yukkysaito @wep21 Any thoughts?

Reply:
@kenji-miyake I agree with the latter because currently we don't need the python library for autoware.

Reply:
@xmfcx Can I remove the python library from the manual installation steps?

Reply:
It's part of the default nvidia tensorrt installation process. Why not keep it?

If someone wants to use it, they could get confused.

Reply:
If you are really worried about the disk-space, you can remove it.

Reply:
> Why not keep it?

It's just because we don't use it now.
If someone wants it in the future, then it's better to review the use case and add it.
I'm worried about some unintended dependencies will be added, it could be troublesome.

But I don't have so strong opinion on this. It's enough if we can make Ansible and Manual consistent.

@xmfcx So considering my answer, which do you prefer? I'll leave it to you.

Reply:
We can remove it, it's ok.

Reply:
Thank you! Then I'll send a PR to `autowarefoundation/autoware`. After that, please approve this. :pray:

Reply:
https://github.com/autowarefoundation/autoware/pull/177